### DISCUSSION TITLE
Endless trouble

### DISCUSSION THREAD

**Discussion by Canusmellthatbag** [sherlock-project/sherlock#2290] [Q&A]
I have no experience doing anything coding wise but I have been running into nothing but problems even with the assistance of ai, I don't know how to even describe what's happening but I'm trying to successfully install and run holehe, project-sherlick, and maigret,  any help is appreciated I tried on android forever finally got my hands on a laptop figured that was the ticket.... it was not
![IMG_20240909_213837187](https://github.com/user-attachments/assets/1ce6327f-fc42-4350-a97f-3b5fe67e1367)
![IMG_20240909_213645774](https://github.com/user-attachments/assets/30cc152e-c067-4296-89c4-56c2cc10e2c7)
![IMG_20240909_213712562](https://github.com/user-attachments/assets/3106fe78-95b0-4706-a5db-1e49eed15633)
![IMG_20240909_213850943](https://github.com/user-attachments/assets/bb228cee-cbb2-4034-8e4a-6a84268d7235)
![IMG_20240909_213737539](https://github.com/user-attachments/assets/69144822-565c-48e5-af1c-6095f75a829d)
![IMG_20240909_213819049](https://github.com/user-attachments/assets/b1e08563-dd5b-4761-ac18-d0d5e7349ad3)

**Comment by vehbiu** [comment]
There is no trouble here, it is installing the packages which is what it's supposed to do. You just have to run sherlock now.

**Reply by DrCFC** [reply]
python 3 sherlock.py doesnt run

**Reply by vehbiu** [reply]
`python3 -m sherlock` is what you're looking for

**Reply by vehbiu** [reply]
@DrCFC

**Comment by bytexenon** [comment]
Apologies for the delayed response. I've noticed a couple of issues in your screenshots:

1. You posted a discussion in the Sherlock Project repository, - a tool for finding social media accounts by username. However, your screenshots show that you're installing [Holehe](https://github.com/megadose/holehe). You should've opened an issue on [their Issues page instead](https://github.com/megadose/holehe/issues). Be sure to understand the second point before doing so.
2. The issue you're encountering isn't related to any tool. When installing a Python package with `pip` or `pipx`, make sure to specify the command before any arguments. According to the Holehe installation guide, you should install it using `pip3 install holehe` and then run it as `holehe your_email@domain.com`. You mistakenly ran `holehe --version` and entered the email as a separate command, which is incorrect.

> Please consider using a screenshot tool to provide clearer images in the future. On Windows, you can use the built-in tool by pressing Win+PrntScr ("PS" key)
